_health-care/_js/_react-entry.jsx

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
```

Comment:
Let's get OCD about import ordering...

How about

first block: all vendor related imports in alphabetical order (as best as possible).

second block: all our own imports in alphabetical order.  So this would look like:

```
 import ReactDOM from 'react-dom';
 import { IndexLink, Link, Route, Router, hashHistory } from 'react-router';

 import DateInput from './_components/date_input.jsx';
 import MothersMaidenName from './_components/mothers_maiden_name.jsx';
 import VeteranName from './_components/veteran_name.jsx';
```